### PR TITLE
Pud 540 update recall with assessed by user

### PR DESCRIPTION
**Pull request by mikedunfordEE**

### CODE REVIEW

File: server/@types/index.d.ts [hunk 1]
```diff
@@ -144,6 +144,7 @@ export interface EmailUploadValidatorArgs {
   emailFileSelected: boolean
   uploadFailed: boolean
   allowedFileExtensions: string[]
+  actionedByUserId: string
```
Comment (jonnywyatt) [inline server/@types/index.d.ts:147]: Minor but I'd just call this `assessedByUserId` so it matches up with the API property name

File: server/@types/manage-recalls-api/models/AddDocumentRequest.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@ export namespace AddDocumentRequest {
     PART_A_RECALL_REPORT = 'PART_A_RECALL_REPORT',
     PREVIOUS_CONVICTIONS_SHEET = 'PREVIOUS_CONVICTIONS_SHEET',
     PRE_SENTENCING_REPORT = 'PRE_SENTENCING_REPORT',
+    RECALL_NOTIFICATION = 'RECALL_NOTIFICATION',
```
Comment (jonnywyatt) [inline server/@types/manage-recalls-api/models/AddDocumentRequest.ts:19]: Is this the same thing. as `RECALL_NOTIFICATION_EMAIL`, below?

File: server/@types/index.d.ts (hunk 1 above)
Comment (mikedunfordEE) [reply server/@types/index.d.ts:147]: I intentionally didn't call it that as this type is used in 2 places, also when creating the dossier.  Which will presumably have a different API property name

File: server/@types/manage-recalls-api/models/AddDocumentRequest.ts (hunk 2 above)
Comment (mikedunfordEE) [reply server/@types/manage-recalls-api/models/AddDocumentRequest.ts:19]: no idea, this is what happened when I ran swagger-to-ts

Comment (mikedunfordEE) [reply server/@types/manage-recalls-api/models/AddDocumentRequest.ts:19]: Oh no, it is not the same thing.  The 'Recall notification' is now being saved when it is generated

**Pull request merged**